CHANGELOG.md

```diff
@@ -0,0 +1,104 @@
+- [Defaults] Unpopulated Activity and Organisation Schemas may be accessed by overriding a default argument.
+
+- [Exceptions] There is very rudimentary exception handling.
+- [Exceptions] Some exceptions are logged.
+- [Exceptions] Some raised exceptions have descriptions to make it possible to consistently determine the cause.
+
+- [Resources] Resources are static files that come with the library, split into 'The Standard' and 'Test Data'.
+- [Resources] Resources are stored and accessed based on a particular Decimal Version.
+- [Resources] Schemas, Codelists and Rulesets are available for V1.04-2.02.
+- [Resources] Schemas are available for V1.03 and prior.
+- [Resources] Non-embedded Codelists are deemed to have a single form valid for V1.04-2.02.
+- [Resources] Schema test data from the SSOT is present for V1.01-2.02.
+- [Resources] Custom test data has been created for V2.02.
+- [Resources] Resources may be loaded from disk as bytes, a Dataset, a string or an ElementTree.
+
+- [Rulesets] Blank Rulesets may be created by providing no arguments.
+- [Rulesets] Rulesets may be created from a string conforming to the Ruleset Schema.
+- [Rulesets] Rulesets contain a set of Rules.
+- [Rulesets] A Dataset may have truthy conformance checked against a Ruleset.
+- [Rulesets] Rules may be created from a `context` and a `case`.
+- [Rulesets] Each of the Rules in the Ruleset Schema have been implemented for instantiation and truthy conformance. These are: atleast_one, date_order, dependent, no_more_than_one, regex_matches, regex_no_matches, startswith, sum, unique
+- [Rulesets] A Rule base class exists to allow the creation of custom types of Rule.
+- [Rulesets] The string representation of a Rule details what must occur for a Dataset to be conformant.
+- [Rulesets] A Dataset may have truthy conformance checked against a Rule. NOTE: A Rule may skip or raise an error.
+
+- [Schemas] Schemas may be created by providing a path to a file containing a valid XSD.
+- [Schemas] Schemas contain a set of Codelists.
+- [Schemas] Schemas contain a set of Rulesets.
+- [Schemas] Should a XSD be defined across multiple files, the resulting Schema shall be flattened to allow programmatic access to all of its contents.
+- [Schemas] Activity and Organisation Schemas have been implemented.
+
+- [Utility] A new namespace may be added to a Schema.
+- [Utility] There is a function to convert from ElementTree to XMLSchema.
+- [Utility] There is a function to convert from a string to an ElementTree.
+- [Utility] There is a function to work around RFC4627 by preventing duplicate keys in a JSON file.
+- [Utility] There is very rudimentary logging functionality.
+- [Utility] Various functionality to access Version Numbers is present.
+
+- [Compatibility] Compatible with Python 2.7, 3.4, 3.5 and 3.6.
+
+- [Tests] Comprehensively unit test all code.
```

Comment:
`Comprehensively unit test all code.` -> `All code is comprehensively unit tested.`